src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -73,7 +73,8 @@ object CromwellApiService {
 trait CromwellApiService extends HttpService with PerRequestCreator {
   val workflowManager: ActorRef
 
-  val workflowRoutes = queryRoute ~ workflowOutputsRoute ~ submitRoute ~ callOutputsRoute ~ callStdoutStderrRoute ~ abortRoute
+  val workflowRoutes = queryRoute ~ workflowOutputsRoute ~ submitRoute ~ workflowStdoutStderrRoute ~ abortRoute ~
```

Comment:
I know we copied this from someone else around here but we can just have a single routes thing instead of forgetting to add stuff. I don't really care if that gets resolved here though, but I know it bit you